pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -194,8 +203,28 @@ export const ApplicationFormTabs: React.FC<ApplicationFormProps> = (props) => {
           onChange={handleChange}
           aria-label="basic tabs example"
         >
-          <Tab label="Add manually" {...a11yProps(0)} />
-          <Tab label="Add from Git (Alpha)" {...a11yProps(1)} />
+          <Tab
+            classes={{ wrapper: classes.tab, labelIcon: classes.labelIcon }}
+            label="Add manually"
+            icon=" "
```

Comment:
The labels in the tabs are slightly misaligned between left and right tabs, so I added a blank icon to the left, I know it's quite dirty though.